### PR TITLE
New Package: Schezo.Lhaplus

### PR DESCRIPTION
**Pull request by Kyure-A**
- 

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/67638)

### CODE REVIEW

**Comment by Kyure-A** [comment]
What exactly is meant by "evidence" here?

**Comment by Kyure-A** [comment]
The software is "redistributable as long as the archive file is not modified" as stated in this URL (http://www7a.biglobe.ne.jp/~schezo/ReadMe.txt).

Is the above text acceptable as "evidence"?

File: manifests/s/Schezo/Lhaplus/1.7.4.0/Schezo.Lhaplus.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Schezo.Lhaplus
+PackageVersion: 1.7.4.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://dforest.watch.impress.co.jp/library/l/lhaplus/10418/lpls174.exe
+  InstallerSha256: 142D8F2ED308EFE9E1DCB8657757D99848827C40A5199436EB36C53F2B9D40FE
```
Comment (ItzLevvie) [inline manifests/s/Schezo/Lhaplus/1.7.4.0/Schezo.Lhaplus.installer.yaml:10]:
```suggestion
  InstallerSha256: 142D8F2ED308EFE9E1DCB8657757D99848827C40A5199436EB36C53F2B9D40FE
  ProductCode: Lhaplus
```

File: manifests/s/Schezo/Lhaplus/1.7.4.0/Schezo.Lhaplus.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Schezo.Lhaplus
+PackageVersion: 1.7.4.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://dforest.watch.impress.co.jp/library/l/lhaplus/10418/lpls174.exe
```
Comment (ItzLevvie) [inline manifests/s/Schezo/Lhaplus/1.7.4.0/Schezo.Lhaplus.installer.yaml:9]:
```suggestion
  InstallerUrl: http://www7a.biglobe.ne.jp/~schezo/lpls174.exe
```

File: manifests/s/Schezo/Lhaplus/1.7.4.0/Schezo.Lhaplus.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Schezo.Lhaplus
+PackageVersion: 1.7.4.0
+PackageLocale: en-US
+Publisher: Schezo
+PackageName: Lhaplus
+License: http://www7a.biglobe.ne.jp/~schezo/ReadMe.txt
```
Comment (ItzLevvie) [inline manifests/s/Schezo/Lhaplus/1.7.4.0/Schezo.Lhaplus.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
LicenseUrl: http://www7a.biglobe.ne.jp/~schezo/ReadMe.txt
PackageUrl: http://www7a.biglobe.ne.jp/~schezo/
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Kyure-A** [comment]
why

**Comment by ItzLevvie** [comment]
Wait for an engineer to take a look.

**Comment by Kyure-A** [comment]
Thank you.

**Pull request merged**